internal/controllers/port/tests/port-create-minimal/00-port.yaml

```
@@ -0,0 +1,14 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Port
+metadata:
+  name: port-create-minimal
+spec:
+  cloudCredentialsRef:
+    cloudName: openstack
+    secretName: openstack-clouds
+  managementPolicy: managed
+  networkRef: port-create-minimal
+  resource:
+    addresses:
+      - subnetRef: port-create-minimal
```

Actually, as we discussed this morning, we should be able to create a port without subnet attachment. 

In this case, we should create the port with an empty resource spec:
```suggestion
  resource: {}
```